custom/enikshay/tasks.py

```diff
@@ -0,0 +1,109 @@
+import datetime
+from collections import defaultdict
+from django.utils.dateparse import parse_datetime, parse_date
+
+from .case_utils import get_adherence_cases_between_dates_from_episode
+from corehq.apps.fixtures.models import FixtureDataItem
+
+
+DOMAIN = 'enikshay'
+DOSE_TAKEN_INDICATORS = [
+    'directly_observed_dose',
+    'unobserved_dose',
+    'self_administered_dos',
+]
+DAILY_SCHEDULE_ID = 'schedule_daily'
+CASE_TYPE_PERSON = 'person'
+
+
+def get_doses_data():
+    # return 'doses_per_week' by 'schedule_id' from the Fixture data
+    fixtures = FixtureDataItem.get_indexed_items(DOMAIN, DAILY_SCHEDULE_ID, 'schedule_id')
+    return dict((k, int(fixture['doses_per_week'])) for k, fixture in fixtures.items())
+
+
+def get_open_episode_cases():
```

Comment:
Sorry, I meant to do `get_open_case_ids_in_domain_by_type(CASE_TYPE_EPISODE)`. Not by `CASE_TYPE_PERSON`.